src/rpc/grpc.md

```diff
@@ -0,0 +1,40 @@
+# gRPC
+
+Sitting on top of HTTP/2 and protobuf gRPC provides a high performance RPC
+framework. It supports features such as bi-directional streaming and authentication.
+
+gRPC is also part of the [CNCF](https://www.cncf.io/projects/). It is commonly
+used in applications like Kubernetes or TiKV.
+
+## Servers and Stubs
+
+gRPC works off the idea that there is a *server* node which can accept
+*requests* from *stub* client nodes.

```

Comment:
I think it is actual clients which send request, not client stubs, the stubs are generated by the gRPC library and then filled out by the programmer, but the stubs alone don't do much.